src/linear_algebra/eigenspace.lean

```diff
@@ -33,8 +34,8 @@ notation for it.
 
 ## References
 
-* [Sheldon Axler, *Down with determinants!*,
-  https://www.maa.org/sites/default/files/pdf/awards/Axler-Ford-1996.pdf][axler1996]
+* [Sheldon Axler, *Linear Algebra Done Right*,
+  https://zhangyk8.github.io/teaching/file_spring2018/linear_algebra_done_right.pdf][axler2015]
```

Comment:
This doesn't look like a legal copy of the book so we probably shouldn't add it to mathlib. Also, an entry for `axler2015` should be added to `docs/references.bib`.